cockatrice/src/dialogs/dlg_settings.cpp

```diff
@@ -544,6 +558,9 @@ void UserInterfaceSettingsPage::retranslateUi()
     tapAnimationCheckBox.setText(tr("&Tap/untap animation"));
     deckEditorGroupBox->setTitle(tr("Deck editor settings"));
     openDeckInNewTabCheckBox.setText(tr("Always open deck in new tab"));
+    replayGroupBox->setTitle(tr("Replay settings"));
+    rewindBufferingMsLabel.setText(tr("Amount of time to buffer a backwards skip from keyboard shortcut by:"));
```

Comment:
someone please think of a better name for this

Reply:
`Buffer time for backwards skip via shortcut` ?